jekyll/_cci2/basics.md

```diff
@@ -7,6 +7,35 @@ description: "Basics"
 
 Get started by learning about the basic concepts used in CircleCI.
 
+### Auto Cancel a Redundant Build
```

Comment:
Since this doesn't work for workflows, I think we should move it to the Skip a Build doc and modify that title to include 'Cancel' in the title.

Reply:
I thought landing pages were good places to put specific documentation about features? :)

https://circleci.com/docs/2.0/test/